### ISSUE TITLE
no-ex-assign: Rule is triggered outside of catch blocks

### ISSUE THREAD

**Issue by nfriend** [buzinas/tslint-eslint-rules#62]
The following causes a linting error when `no-ex-assign` is set to `true` (line 4, column 12):

```typescript
class MyClass {}

interface IMyInterface {
    myVar: MyClass;
}
```

**Comment by buzinas** [comment]
Similar to #54, I'll close this, and keep the tracking there.
